storage/innobase/xtrabackup/test/t/xb_export.sh

```diff
@@ -64,12 +64,28 @@ function insert_data() {
 }
 
 function create_tables() {
-  ### check if we can import all the columns type in mysql
```

Comment:
we already have datetime here, why didn't this test fail?
I guess, that could be because, we only care about the number of .cfg files created but didn't do actual import? :thinking:

Reply:
It didnot fail because there was no scenario for testing temporal columns with different collations.

Reply:
No. Explicit collation given at the column declaration are ignored by server for temporal columns.

Reply:
ah, so ALTER is compulsory?

Reply:
Yes. Only through ALTER TABLE COVERT TO CHARACTER SET server allowed collation to change. However we are disallowing in server it as part of PS-9219. So MySQL won't be able to change the charsets of temporal types.